news/4187.feature

```diff
@@ -0,0 +1,5 @@
+Allow PEP 508 URL requirements to be used as dependencies.
+
+As a security measure, all packages installed from PyPI are specifically blacklisted
```

Comment:
Minor preference to not say "blacklisted" here. I'd rather just say:

> As a security measure, this is not supported when installing from PyPI for dependencies not hosted on PyPI. In the future, PyPI will block uploading packages with such external URL dependencies directly.

Reply:
I'm trying to point attention to the fact that this is a conscious choice to make it not work for a specific reason, rather than just unsupported which felt more to me like "may or may not work."

Does that make sense?

Reply:
Indeed. My reservation here is that this says "blacklist" here (and in the comment) but we don't actually have a "list" of packages that we disallow based on -- it's on a characteristic of how the installation is being done.

So, a phrasing that doesn't use "blacklist" is something I'd prefer but I won't block this PR for this. :P